src/pyBiodatafuse/annotators/wikidata.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import datetime
+import os
+from string import Template
+
+import pandas as pd
+from SPARQLWrapper import JSON, SPARQLWrapper
+
+from pyBiodatafuse.utils import collapse_data_sources, get_identifier_of_interest
+
+
+def get_version_wikidata() -> dict:
+
+    wikidata_version = {"wikidata_version": "0"}
```

Comment:
I think wikidata doesn't provide version, but is it a common practice to use 0?

Reply:
No, was just a template. I am not sure Wikidata has a verion. it changes all the time. I'll come back on this.

Reply:
I'm using `str(datetime.datetime.now())[0:16]` now.